docs/getting-started/security-spaces.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[security-spaces]]
+= Spaces and {elastic-sec}
+
+{elastic-sec} supports the organization of your security operations into
+logical instances with the {kibana-ref}/xpack-spaces.html[{kib} spaces]
+feature. Each space in {kib} represents a separate logical instance of
+{elastic-sec} in which detection rules, rule exceptions, value lists,
+alerts, Timelines, cases, and {kib} advanced settings are private to the
+space and accessible only by users that have role privileges to
+access the space. For details about configuring privileges for
+{es} and {kib}, refer to <<detections-permissions-section>>.
+
+For example, if you create a `SOC_prod` space in which you load and
+activate all the {elastic-sec} prebuilt detection rules, these rules and
+any detection alerts they generate will be accessible only when visiting
+the {security-app} in the `SOC_prod` space. If you then create a new
+`SOC_dev` space, you'll notice that no detection rules or alerts are
+present. Any rules subsequently loaded or created here will be private to
+the `SOC_dev` space, and they will run independently of those in the
+`SOC_prod` space.
+
+[NOTE]
+===== 
+By default, alerts created by detection rules are stored in {es} indices
+under the `.siem-signals-<Kibana-space>-*` index pattern, and they may be
+accessed by any user with role privileges to access those
+{es} indices. In our example above, any user with {es} privileges to access
+`.siem-signals-SOC_prod-*` will be able to view `SOC_prod` alerts from
+within {es} and other {kib} apps such as Discover and Lens. 
```

Comment:
++ Thanks for the clarification here! Was my only remaining question after reading the above.